internal/safeconfig/safeconfig.go

```diff
@@ -0,0 +1,50 @@
+package safeconfig
```

Comment:
I would honestly leave this whole thing out... don see the problem in restarting the exporter on config changes...

Reply:
Sorry, didn't understand this point. Can you expand it a bit?
This is package is describe config which will be loaded from file.
If config file will change, it's necessary to restart exporter.

Reply:
oh, my bad, I read it wrong, sorry 😂